theme/style.css

```diff
@@ -16,3 +16,16 @@
 }
 
 /* style your grid sections here!  */
+
+#nate{
```

Comment:
you have a `class` of `nate`, but this ☝️ selector is selecting an `id` called `nate`